app/src/main/java/fr/free/nrw/commons/navtab/MoreBottomSheetFragment.kt

```diff
@@ -182,14 +182,17 @@ class MoreBottomSheetFragment : BottomSheetDialogFragment() {
         Single.defer { single }
             .subscribeOn(Schedulers.io())
             .observeOn(AndroidSchedulers.mainThread())
-            .subscribe { success ->
+            .subscribe({ success ->
                 val messageResId = if (success) {
                     R.string.thanks_feedback
                 } else {
                     R.string.error_feedback
                 }
                 Toast.makeText(requireContext(), getString(messageResId), Toast.LENGTH_SHORT).show()
-            }
+            }, { error ->
+                error.printStackTrace()
+                Toast.makeText(requireContext(), R.string.error_feedback, Toast.LENGTH_SHORT).show()
```

Comment:
same

Reply:
I guess one toast is left, can you change it to snackbar?

Reply:
This was there earlier as well. Should I change it to Snackbar

Reply:
Its okay for now.